### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by cyrus0111** [vchlum/hue-lights#82] [Q&A]
Hey ho,
i'm new in using linux so how can i install this package?
thx for answering

**Comment by vchlum** [comment]
Hi @cyrus0111 !
* To install this extension make sure you are using Gnome GUI. e.g.  It is the default GUI in Ubuntu. Using other GUI, the extension can not be used.
* Using Gnome, the easiest way to install it, is from [this link](https://extensions.gnome.org/extension/3737/hue-lights/). 
* First time on the link, you should be asked to install the page's extension into your [Chrome](https://chromewebstore.google.com/detail/gnome-shell-integration/gphhapmejobijbbhgpjhcjognlahblep) or Firefox.  Just follow the directions on the page.
* After installing the browser extension, you should see an install button on [the page with hue-lights extension](https://extensions.gnome.org/extension/3737/hue-lights/).
